index.bs

```diff
@@ -1371,6 +1371,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 </div>
 
+###  Preventing silent access to an existing credential - PublicKeyCredential's `[[ preventSilentAccess]](credential, sameOriginWithAncestors)` method ### {# preventSilentAccessCredential}
+
+<div link-for-hint="PublicKeyCredential/[[ preventSilentAccess]](credential, sameOriginWithAncestors)">
+
+Calling the <dfn for="PublicKeyCredential" method>\[[ preventSilentAccess]](credential, sameOriginWithAncestors)</dfn> method
```

Comment:
Should there be a space in `[[ preventSilentAccess]]`? There's also a `\` just before it which shows up in the rendered text.